src/Common/src/TypeSystem/Common/WellKnownType.cs

```diff
@@ -40,5 +40,6 @@ public enum WellKnownType
         RuntimeFieldHandle,
 
         Exception,
+        IndexOutOfRangeException,
```

Comment:
My line of thinking was that this is one of the rare types defined by the runtime portion of ECMA and it's therefore fine.

We don't have other good ways to get to it (besides assuming it's in the core library and look for it by name).